manifest.json

```diff
@@ -1417,6 +1417,13 @@
       "type": "library",
       "remote": "https://github.com/adamharrison/lite-xl-www.git:aecc866c33b0a954fbd41a5de2842bb97f4ac45e",
       "mod_version": "3"
+    },
+    {
+     "id": "code+",
```

Comment:
I don't know if you can have symbols in the `id`, but you can for sure in the `name` field.
@adamharrison could you confirm?

Reply:
Thank you, I will make a new commit with the suggested changes. cheers

Reply:
Confirmed, ids should be `^[a-z0-9\-_]+$` as per [SPEC](https://github.com/lite-xl/lite-xl-plugin-manager/blob/master/SPEC.md).